app/models/project.rb

```diff
@@ -0,0 +1,41 @@
+class Project
+  attr_accessor :user, :pledge
+  attr_reader :pledge_goal_amount
+  @@all = []
+  def initialize(user, pledge_goal_amount)
+    @user = user
+    @pledge = []
+    @pledge_goal_amount = pledge_goal_amount
+    @@all << self
+  end
+
+  def self.all
+    @@all
+  end
+
+  def add_pledge(pledge)
+    @pledge << pledge
```

Comment:
tie your add_pledge to your Pledge class by initializing a Pledge obj. with .new